### DISCUSSION TITLE
Does RSX offer a way to _add_ a CSS classes conditionally?

### DISCUSSION THREAD

**Discussion by bluenote10** [DioxusLabs/dioxus#3092] [Q&A] [answered]
I know that RSX allows something like this:

```rust
rsx! { 
    div { 
        class: if some_signal.get() { "some-class" },
        "Hello, World!" 
    }
}
```

However, what about the case when there are some common classes that should be extended (probably fairly common for tailwind users). This doesn't seem to compile, and so far I couldn't figure out a good solution:

```rust
rsx! { 
    div { 
        class: "some common classes " + if some_signal.get() { "activated-class" }
        "Hello, World!" 
    }
}
```

**Comment by ealmloff** [comment, marked as answer]
Yes, you can add a second `class` attribute that is optional. If they are both true, they will get merged automatically:
```rust
rsx! { 
    div { 
        class: "some common classes ",
        class: if some_signal.get() { "activated-class" }
        "Hello, World!" 
    }
}
```
